scene/gui/base_button.cpp

```diff
@@ -512,6 +524,7 @@ void BaseButton::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "toggle_mode"), "set_toggle_mode", "is_toggle_mode");
 	ADD_PROPERTYNZ(PropertyInfo(Variant::BOOL, "pressed"), "set_pressed", "is_pressed");
 	ADD_PROPERTYNO(PropertyInfo(Variant::INT, "action_mode", PROPERTY_HINT_ENUM, "Button Press,Button Release"), "set_action_mode", "get_action_mode");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "button_mask"), "set_button_mask", "get_button_mask");
```

Comment:
`PROPERTY_HINT_FLAGS` would be nice to have on this property. Also, I think it might be made an`ADD_PROPERTYNO`, so it won't save a value of "1" to disk.

Reply:
Excellent suggestion, thanks ! It looks much better now.